package/network/services/ppp/patches/400-simplify_kernel_checks.patch

```diff
@@ -29,7 +29,7 @@ Signed-off-by: Jo-Philipp Wich <jo@mein.io>
  
```

Comment:
We should update the kernel version from `2,6,37` to something more recent like 5.15 or 6.6. There are some checks for kernel `if (kernel_version >= KVERSION(4,7,0))`.

I think we can drop this complete patch. How much bytes does it save?

Reply:
It only suppresses some error messages which indicate real errors on recent kernel versions.

Reply:
This patch can reduce the size of the package `ppp` by approximately 2.5 KB. I'll rewrite it to remove more useless checks later.